notebooks/util.py

```diff
@@ -362,7 +362,7 @@ def descriptors_by_fragments(fragment_library):
     fragment_library : dict of pandas.DataFrame
         Fragment details, i.e. SMILES, and fragment RDKit molecules, KLIFS and fragmentation details (values)
         for each subpocket (key).
-        
+        Here only value 'smiles' is necessary.
```

Comment:
SMILES and ROMol are necessary in the current function setup. 

We could also only input only SMILES here and calculate the RDKit molecules inside the function, however most functions in `util.py` currently take the library DataFrame as input parameter.

Let me check if I can make the input for all functions more specific.